study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EmailBuilder.java

```diff
@@ -192,10 +183,14 @@ private String readRawHtmlContent(File file) {
         }
     }
 
-    private String renderHtmlContent(String name, String rawHtml) {
+    private String renderHtmlContent(File file, String rawHtml) {
+        VelocityEngine engine = new VelocityEngine();
+        engine.addProperty(RuntimeConstants.RUNTIME_REFERENCES_STRICT, true);
+        engine.addProperty(RuntimeConstants.FILE_RESOURCE_LOADER_PATH, file.getParent());
```

Comment:
Instead of using a single Velocity engine, we're now `new`-ing a new instance on every call. This is needed to allow doing "includes" of files in same directory as `file`, enabling us to do nicer templating.